install-oracle.sh

```diff
@@ -1123,42 +1123,59 @@ echo -e "Running with parameters from command line or environment variables:\n"
 set | grep -E '^(ORA_|BACKUP_|GCS_|ARCHIVE_|INSTANCE_|PB_|ANSIBLE_|CLUSTER|PRIMARY)' | grep -v '_PARAM='
 echo
 
-ANSIBLE_PARAMS="-i ${INVENTORY_FILE} ${ANSIBLE_PARAMS}"
-ANSIBLE_EXTRA_PARAMS="${*}"
+export ANSIBLE_NOCOWS=1
```

Comment:
What do you think about putting it in the ansible.cfg file rather than the shell script?  Are there security concerns with that?